docs/tutorials/talk-to-kafka.rst

```diff
@@ -0,0 +1,688 @@
+
+Learning objectives
+--------------------
+
+- Basic overview of Apache Kafka and creating a highly available Apache Kafka service
+- The concept of topic, partition, producer, consumer, and consumer groups 
+- The need for data serialization in Kafka and use of Apache Avro™ to produce and consume messages
+
+Overview
+--------
+
+Getting started with Apache Kafka can be straightforward: you download the binary and start the bootstrap server. However, getting a production-ready Kafka cluster stand up with security and high-availability is a different story. 
+If your goal is to use Kafka to benefit your application or data needs and NOT to learn Kafka administration, you can consider using `a managed Kafka service <https://aiven.io/kafka>`_, allowing you to start producing and consuming data in minutes.
+
+In this tutorial, we will learn how to create a highly available Apache Kafka service, go over the common tasks of producing and consuming messages, and finally use the popular Apache Avro specification to communicate with your Kafka service. 
+This tutorial will use Python programming language.
+Under the hood, the Python library will make use of the `Producer API <https://kafka.apache.org/documentation>`_ and the `Consumer API <https://kafka.apache.org/documentation>`_.
+
+Prerequisites
+-------------
+
+To get started, you'll need:
+
+- A Kafka service - either local or managed 
+- `Python installed <https://www.python.org/downloads/>`_
+- `Kafka-python <https://github.com/dpkp/kafka-python>`_ library installed, you can install it with the following command:
+
+  .. code:: bash
+
+    pip install kafka-python
+
+- `confluent-Kafka <https://github.com/confluentinc/confluent-kafka-python>`_ library installed that you'll use to produce messages in AVRO format, you can install it with the following command:  
+
+  .. code:: bash
+
+    pip install confluent-kafka
+
+- `fastavro <https://github.com/fastavro/fastavro>`_ library installed, you can install it with the following command:
+
+  .. code:: bash
+
+    pip3 install fastavro
+    
+If you already have a Kafka service, jump to :ref:`tutorial-kaka-python-create-a-topic` section. 
+
+Else, `sign up for an Aiven account <https://console.aiven.io/signup>`_ and follow the tutorial to create a Kafka service on Aiven.
+
+Create an Aiven for Apache Kafka® service
+-----------------------------------------
+
+To create an :doc:`Aiven for Apache Kafka </docs/products/kafka>` service, you need to:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+2. On the *Services* page, click **Create a new service**.
+
+   This opens a new page with the available service options.
+
+   .. image:: /images/platform/concepts/console_create_service.png
+      :alt: Aiven Console view for creating a new service
+
+3. Select **Apache Kafka®**.
+
+4. Select the cloud provider and region that you want to run your service on.
+
+5. Select `business-4` as service plan.
```

Comment:
Why do we need a `business-4`? If there's a particular requirement (like being able to enable Karapace) we should mention, otherwise we should select a minor plan